docs/logs/extending-the-sdk/Program.cs

```diff
@@ -53,6 +55,13 @@ public static void Main()
 
         // message will be redacted by MyRedactionProcessor
         logger.LogInformation("OpenTelemetry {sensitiveString}.", "<secret>");
+
+        // Attempt to flush the remaining logs when using an exporter that may require it, such as the custom one defined here.
```

Comment:
I suspect this is not exactly what you wanted @reyang but hopefully it's closer.

Reply:
Definitely moving closer, what do you think regarding https://github.com/open-telemetry/opentelemetry-dotnet/pull/6012#pullrequestreview-2485611201? I want to gather ideas from folks here.

Reply:
Sorry, I'm not super familiar with github PRs and I didn't have a reply option there - let me try to respond there as well.